index.html

```diff
@@ -192,12 +192,12 @@
 }
 @font-face {
     font-family: 'OpenMojiBlack';
-    src: url('font/OpenMoji-Black.ttf') format('truetype');
-}
+    src: url('font/OpenMoji-black-glyf/OpenMoji-black-glyf.woff2') format('woff2');
+  }
```

Comment:
Tiny nit: you forgot to regenerate `index.html` after the change in the generator script :)

Reply:
Done